carta/cpp/core/ScriptedClient/ScriptFacade.cpp

```diff
@@ -768,10 +772,15 @@ QStringList ScriptFacade::getImageDimensions( const QString& controlId ) {
     if ( obj != nullptr ){
         Carta::Data::Controller* controller = dynamic_cast<Carta::Data::Controller*>(obj);
         if ( controller != nullptr ){
-            resultList = controller->getImageDimensions( );
-            if ( resultList[0] == "" ) {
+            std::vector<int> dimensions = controller->getImageDimensions();
+            if ( dimensions.size() == 1 && dimensions[0] == 0 ) {
```

Comment:
You seem to check for a special case where dimensions are [0]. What about [], or [0,0], or [0,0,0....0]? Are those possible?